client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,67 @@
+.scoreboard > div.scoreboardIntermediate {
+  border: 2px solid black;
+  box-sizing: border-box;
+  background: #fff;
+  border-width: 2px;
+  border-radius: 8px;
+  justify-content: center;
+  align-items: center;
+  overflow: auto;
+  height: 100%;
+  width: 100%;
+}
+
+.scoreboard > div.scoreboardIntermediate > table {
+  border-collapse: separate;
+  border-spacing: 0;
+  width: 100%;
+}
+
+.scoreboard > div.scoreboardIntermediate > table tbody td {
+  border-bottom: 1px solid #ddd;
+  border-right: 1px solid #ddd;
+  color: #333;
+  padding: 5px;
+  text-align: right;
+  white-space: nowrap;
+  background-color: #f8f8f8;
+}
+
+.scoreboard.equalWidth > div.scoreboardIntermediate > table td {
+  white-space: normal;
+  width: calc((100% - var(--firstColWidth)) / (var(--columns) - 1));
```

Comment:
This may be the right answer, but then something needs to be done about the first column.  Currently it looks like this:

![Current](https://user-images.githubusercontent.com/76912527/209451644-de04bbdc-d9ba-4eed-a7b3-367c313db21e.png)

With this change, it looks like this:

![Proposed](https://user-images.githubusercontent.com/76912527/209451648-5c6e13c1-d7c0-44c7-aa43-e60e08d33bc4.png)

Again, with default values, the first column will display Round and Totals.  They should fit on one line.

Reply:
That's the white space thing that is set to break all